api/src/orgs/data.js

```diff
@@ -30,10 +30,35 @@ export function getOrganisations (
     )
   }
 
+  const query = db.collection('orgs')
   return makeConnection(
-    db.collection('orgs'),
+    query,
     args,
     filter,
     camelToSnakeCaseKeys(args.sort)
   )
+    .then(augmentWithOrgStats(query, { ...filter }))
+}
+
+const augmentWithOrgStats = (query, filter) => async function (obj) {
+  return {
+    ...obj,
+    ...(await makeOrgStats(query, filter))
+  }
+}
+
+async function makeOrgStats (query, filter) {
+  const [{ totalAUM, totalActivity }] = await query.aggregate(
+    [
+      { $match: filter },
+      {
+        $group: {
+          _id: null,
+          totalAUM: { $sum: '$aum' },
+          totalActivity: { $sum: '$activity' }
+        }
+      }
+    ]
+  ).toArray()
```

Comment:
```suggestion
  const { totalAUM, totalActivity } = await query.aggregate(
    [
      { $match: filter },
      {
        $group: {
          _id: null,
          totalAUM: { $sum: '$aum' },
          totalActivity: { $sum: '$activity' }
        }
      }
    ]
  ).next()
```

Reply:
Also it would be nice if you could put the opening/closing brackets of the aggregation pipeline array on the same lines of the opening/closing parenthesis of the `Cursor#aggregate` function call.